testsuite/integration/basic/src/test/java/org/wildfly/test/integration/observability/opentelemetry/RestClientTestCase.java

```diff
@@ -27,21 +27,18 @@
 import javax.ws.rs.client.Client;
 import javax.ws.rs.client.ClientBuilder;
 
-import org.jboss.arquillian.container.test.api.Deployment;
-import org.jboss.arquillian.junit.Arquillian;
 import org.jboss.as.test.shared.TestSuiteEnvironment;
 import org.jboss.shrinkwrap.api.Archive;
 import org.jboss.shrinkwrap.api.ShrinkWrap;
 import org.jboss.shrinkwrap.api.asset.EmptyAsset;
 import org.jboss.shrinkwrap.api.spec.WebArchive;
-import org.junit.Test;
-import org.junit.runner.RunWith;
 import org.wildfly.common.Assert;
 
-@RunWith(Arquillian.class)
+//@RunWith(Arquillian.class)
+//@ServerSetup(OpenTelemetrySetupTask.class)
```

Comment:
Don't the changes here basically disable or otherwise break this test?

Reply:
That test was supposed to be deleted, as I merged it into the other one so we're not starting and stopping the server twice.